2021-09-09 13:15:00,258 INFO    MainThread:41536 [internal.py:wandb_internal():86] W&B internal server running at pid: 41536, started at: 2021-09-09 13:15:00.247734
2021-09-09 13:15:00,265 INFO    WriterThread:41536 [datastore.py:open_for_write():77] open: C:\Programming\rl_cooperation\Pygame_RL_cooperation\Algorithm\DQN_SpaceCannons_PER\wandb\offline-run-20210909_131458-1dtdjys4\run-1dtdjys4.wandb
2021-09-09 13:15:00,266 DEBUG   HandlerThread:41536 [handler.py:handle_request():126] handle_request: run_start
2021-09-09 13:15:00,288 DEBUG   HandlerThread:41536 [meta.py:__init__():40] meta init
2021-09-09 13:15:00,288 DEBUG   HandlerThread:41536 [meta.py:__init__():54] meta init done
2021-09-09 13:15:00,288 DEBUG   HandlerThread:41536 [meta.py:probe():211] probe
2021-09-09 13:15:00,379 DEBUG   HandlerThread:41536 [git.py:repo():33] git repository is invalid
2021-09-09 13:15:00,380 DEBUG   HandlerThread:41536 [meta.py:_save_pip():58] save pip
2021-09-09 13:15:00,382 DEBUG   HandlerThread:41536 [meta.py:_save_pip():72] save pip done
2021-09-09 13:15:00,383 DEBUG   HandlerThread:41536 [meta.py:probe():249] probe done
